components/gitpod-cli/cmd/tasks.go

```diff
@@ -13,10 +13,11 @@ var tasksCmd = &cobra.Command{
 	Use:   "tasks",
 	Short: "Interact with workspace tasks",
 	Args:  cobra.NoArgs,
```

Comment:
I wonder should not we remove `Args` and `RunE` here alltogether? And the same for ports? It seems we don't have it for timeout and it works the same way?

The only benefit I see that we learn when a user using `gp tasks` from analytics. But maybe then we should add the same for `gp timeout`?